### ISSUE TITLE
Is there a reason, the conversion properties return double for quantities with base type decimal?

### ISSUE THREAD

**Issue by pgrawehr** [angularsen/UnitsNet#1058]
Some quantities, for instance `information` have `decimal` as base type (which does make sense). However, except for the `Value` property, all properties return double:

```csharp
/// <summary>
///     Gets a <see cref="double"/> value of this quantity converted into <see cref="InformationUnit.Bit"/>
/// </summary>
public double Bits => As(InformationUnit.Bit);

/// <summary>
///     Gets a <see cref="double"/> value of this quantity converted into <see cref="InformationUnit.Byte"/>
/// </summary>
public double Bytes => As(InformationUnit.Byte);
```

This is a bit confusing and unnecessarily looses precision. Would it be possible to change these properties to `decimal`, too?

**Comment by tmilnthorp** [comment]
I agree it's not great.

We just closed #1051 which would have deprecated the double conversion properties in favor of returning a Quantity type. You would then be able to use the ```Value``` property (or ```IConvertible``` interfaces) to retrieve the result as a ```decimal```.

Feel free to look over the arguments there, but I do feel that returning a ```Quantity``` gives you more power/flexibility in terms of what you can do with the result.

**Comment by tmilnthorp** [comment]
Although not as convenient, you can also use
```csharp
BitRate rate = BitRate.FromKilobitsPerSecond(56);
decimal mb = rate.ToUnit(BitRateUnit.MegabitPerSecond).Value;
```

**Comment by angularsen** [comment]
Closing this in favor of #875

**Comment by angularsen** [comment]
Re-opening, realized this describes one of several solutions discussed in #875 .
- `IQuantity.Value` should return `QuantityValue`
- `decimal` based quantities should return `decimal` in their conversion properties, like `Information.Bits`

**Comment by angularsen** [comment]
Not stale, PR #1074 recently had activity.

**Comment by pgrawehr** [comment]
Not stale, PR still up for review.

**Comment by angularsen** [comment]
Fixed by #1074
